Readability-readerable.js

```diff
@@ -40,7 +40,7 @@ function isNodeVisible(node) {
  *
  * @return boolean Whether or not we suspect Readability.parse() will suceeed at returning an article object.
  */
-function isProbablyReaderable(doc, isVisible) {
+function isProbablyReaderable(doc, isVisible = null, options = { minScore: 20, minContentLength: 140 }) {
```

Comment:
The way I understand argument default values (and it's past midnight so I'm probably missing something) this requires that a consumer who provides one also provides another. Is that right? That seems counterintuitive.

Would it make sense to instead use `{}` here but use `Object.assign` or something to ensure the defaults are respected if not passed?

Reply:
You're right, that's a very good point. Thanks for the review, I will do that change 👍.
Btw, where is the good place to document these defaults? If we have them in the function signature, that's already somehow documented, but if we move them away, would you prefer to have them in the doc comment, or somewhere else (such as the README file)?

Reply:
Considering the `Object.assign` makes me realize that I can merge both parameters `isVisible` and `options` into one. That way we keep backward compatibility and also avoid to have callers passing a `null` parameter if they just want to change the options.
Something like this:

```js
function isProbablyReaderable(doc, options = {}) {
  // For backward compatibility reasons 'options' can either be a configuration object or the function used
  // to determine if a node is visible.
  if (typeof options === "function") {
    options = {visibilityChecker: options}
  }

 // Here I'm also renaming `isVisible` to `visibilityChecker`. That sounds more like a function, where `isVisible` can be confused for a boolean.
  var defaultOptions = { minScore: 20, minContentLength: 140, visibilityChecker: isNodeVisible }
  options = Object.assign(options || {}, defaultOptions)

  // ...
}
```

I believe that's an improvement over my initial version, though we should be sure to document it correctly (`isVisible` wasn't really documented anywhere anyway, but still).

Reply:
Yep, this suggestion looks great, thanks! Will review the rest in a bit.